libs/service-portal/information/src/lib/messages.ts

```diff
@@ -51,7 +51,7 @@ export const spmm = defineMessages({
     defaultMessage:
       'Íslendingur með lögheimili á Íslandi (t.d. námsmaður eða sendiráðsmaður); í hjúskap með útlendingi sem ekki er á skrá',
   },
-  maritalStatusMarried: {
+  maritalþStatusMarried: {
```

Comment:
```suggestion
  maritalStatusMarried: {
```
I'm guessing this was unintended